README.md

```diff
@@ -436,7 +436,8 @@ All these projects contain steps and resources required for reproduction.
 - [Electromaker micro:bit projects](https://www.electromaker.io/projects?platform=microbit) - All the micro:bit projects posted to Electromaker, a platform for makers to showcase their projects.
 - [Saturday Science & BBC micro:bits](https://saturdayscience.org/bbc-microbit/) - Practical science and engineering projects with the micro:bit, explore physical properties with cool experiments.
 - [Maker Pro micro:bit Projects & Tutorials](https://maker.pro/microbit) - The micro:bit section of Maker Pro, a place for makers to share designs, collaborate, and learn how to take your product to market.
-
+- [microbit-node-and-spotify(Mac)](https://www.hackster.io/samelhusseini/micro-bit-spotify-mac-5ac6d7) - Allows you to control a Spotify playlist with 2 micro:bits on Mac.
+- [microbit-node-and-spotify(Windows)](https://www.hackster.io/samelhusseini/microbit-spotify-windows-8a8551) - Allows you to control a Spotify playlist with 2 micro:bits on Windows.
```

Comment:
As mentioned in https://github.com/carlosperate/awesome-microbit/pull/368#issuecomment-673533506, could you combine these two into one?

Reply:
I'll have a try. I id it like that as they have different links.